clusterman/interfaces/cluster_connector.py

```diff
@@ -90,6 +98,10 @@ def get_cluster_total_resources(self) -> ClustermanResources:
     def _get_agent_metadata(self, ip_address: str) -> AgentMetadata:  # pragma: no cover
         pass
 
+    @abstractmethod
+    def _freeze_agent(self, agent_id: str) -> None:
+        pass
```

Comment:
it feels a little weird to have a private abstract method (i.e., a private function that needs to be implemented as part of the public interface for people subclassing this) - maybe if we get rid of the wrapper above we can make this public with the same name?

additionally, thoughts on renaming this as cordon/uncordon? mesos is not long for this world at yelp and we might as well use standard/consistent terminology

Reply:
I am just removing wrapper method, "cordon" (is_cordoned) has other meaning in clusterman, so I wouldn't use it :) . We add this tag to AWS side, and then check and decide whether it is "is_cordoned"